src/Lykke.Sdk/LykkeStarter.cs

```diff
@@ -17,30 +18,41 @@ public static class LykkeStarter
         /// Starts the service.
         /// </summary>
         /// <typeparam name="TStartup">The type of the startup.</typeparam>
-        /// <param name="componentName">Name of the component.</param>
-        public static async Task Start<TStartup>(string componentName)
+        public static Task Start<TStartup>()
             where TStartup : class
         {
-            Console.WriteLine($@"{componentName} version {
-                    Common.AppEnvironment.Version
-                }");
+            return Start<TStartup>(AppEnvironment.Name, 5000);
+        }
 
-            if (Debugger.IsAttached)
-            {
-                Console.WriteLine(@"Is DEBUG");
-            }
-            else
-            {
-                Console.WriteLine(@"Is RELEASE");
-            }
+        /// <summary>
+        /// Starts the service.
+        /// </summary>
+        /// <typeparam name="TStartup">The type of the startup.</typeparam>
+        /// <param name="componentName">Name of the component.</param>
+        public static Task Start<TStartup>(string componentName)
+            where TStartup : class
+        {
+            return Start<TStartup>(componentName, 5000);
+        }
 
-            Console.WriteLine($@"ENV_INFO: {Common.AppEnvironment.EnvInfo}");
+        /// <summary>
+        /// Starts the service.
+        /// </summary>
+        /// <typeparam name="TStartup">The type of the startup.</typeparam>
+        /// <param name="componentName">Name of the component.</param>
+        /// <param name="port">Port that the app is listening to.</param>
+        public static async Task Start<TStartup>(string componentName, int port)
+            where TStartup : class
+        {
+            Console.WriteLine($@"{componentName} version {AppEnvironment.Version}");
+            Console.WriteLine(Debugger.IsAttached ? "DEBUG mode" : "RELEASE mode");
```

Comment:
The meaning of the mode is not clear. With this flag we must write "Debugger Attached"